sandboxed_containers/understanding-sandboxed-containers.adoc

```diff
@@ -16,13 +16,18 @@ toc::[]
 - Ensure proper isolation and sandboxing for testing software.
 - Ensure default resource containment through VM boundaries.
 
-Furthermore, {sandboxed-containers-first} provide an additional option for users to choose from the type of workload they want to run to cover a wide variety of use cases.
+{sandboxed-containers-first} also provides users the ability to choose from the type of workload that they want to run to cover a wide variety of use cases.
+
+You can use the {sandboxed-containers-operator} to manage the lifecycle of sandboxed containers on a cluster. You can use the Operator to perform tasks such as installation and removal, updates, and status monitoring.
```

Comment:
"to manage the lifecycle of sandboxed containers" sounds to me like the operator can start/stop/delete containers. But it doesn't control the sandboxed containers it only installs/uninstalls the runtime that is controlling the containers. I'd just remove the first sentence.

Reply:
mmm, maybe just an overstatement for 4.8 but do you see it as a valid statement for the desired goal?

Reply:
But I agree, we should reflect the install/un-install bit and revisit the "lifecycle" statement in 4.9